torch/_weights_only_unpickler.py

```diff
@@ -92,6 +92,12 @@ def _get_allowed_globals():
         torch.int16,
         torch.int32,
         torch.int64,
+        # quantized dtypes
+        torch.qint8,
+        torch.qint32,
+        torch.quint8,
+        torch.quint4x2,
+        torch.quint2x4,
```

Comment:
We should have that list somewhere shared. Maybe we should re-use `torch.storage._dtype_to_storage_type_map` ?

Reply:
This list only contains dtypes serialized with `rebuild_tensor_v2` added `torch.storage._new_types` for dtypes serialized with `_rebuild_tensor_v3`